website/docs/components/button/partials/guidelines/guidelines.md

```diff
@@ -17,21 +17,29 @@
 
 <Hds::Button @size="large" @text="Large" />
 
-Medium is the preferred size, but use a button size that best fits the UI.
+<Hds::Button @size="large" @isFullWidth={{true}} @text="Full-width button" />
 
-For example:
+!!! Insight
 
-- don't use large buttons in tables.
+**Migrating Tip** 
 
-_Banner (informational):_ Medium buttons are the same size as default Structure buttons. Small buttons are the same size as Compact Structure buttons.
+Medium buttons are the same size as default Structure buttons. Small buttons are the same size as Compact Structure buttons.
```

Comment:
Q: Should "Compact" be capitalized here?

"Small buttons are the same size as **Compact** Structure buttons."

Reply:
That sounds good to me- can you make it a suggestion so I can merge it from here? 👍